sw/airborne/subsystems/navigation/common_nav.h

```diff
@@ -68,6 +68,7 @@ extern void nav_reset_reference(void) __attribute__((unused));
 extern void nav_reset_alt(void) __attribute__((unused));
 extern void nav_update_waypoints_alt(void) __attribute__((unused));
 extern void common_nav_periodic_task_4Hz(void);
+extern float get_time_to_home(void); /* estimated time to home point in seconds */
```

Comment:
The name is not all to clear sine time to home could mean be the time for the rest of flightplan. BTW was there not just   ETA to a way point already? I know it means the Home mode then the rest of the time... I know for lots of users home mode is handy, IMHO the HOME mode is a bit of a strange option, I rather solve it in a flightplan, but that is some other discussion. for now 11 days indeed would be enough, only

Reply:
All things considered, yes a welcomed addition.

Reply:
pprzlink unaligned access, I cannot estimate implications, but assuming you gave it thought.

Reply:
unaligned access was already enabled for stm32, ChibiOS was just missing